prowler/lib/mutelist/mutelist.py

```diff
@@ -237,6 +238,37 @@ def is_muted_in_check(
             )
             return False
 
+    def mute_finding(self, finding):
+        """
+        Check if the provided finding is muted
+
+        Args:
+            finding (Finding): The finding to be evaluated for muting.
+
+        Returns:
+            Finding: The finding with the status updated if it is muted, otherwise the finding is returned
+
+        """
+        try:
+            is_finding_muted = self.is_muted(
+                finding.account_uid,
+                finding.metadata.CheckID,
+                finding.region,
+                finding.resource_id,
+                unroll_dict(unroll_tags(finding.resource_tags)),
+            )
+
+            if is_finding_muted:
+                finding.raw["status"] = finding.status
+                finding.status = "MUTED"
```

Comment:
Please, use the `Status` enum here, add it if not present.

Reply:
Done